app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,34 @@
+class MoviesController < ApplicationController
+
+  def index
+    movies = Movie.all
+    render json:  movies.as_json(only: [:id, :title, :release_date]), status: :ok
+  end
+
+  def show
+
+    movie = Movie.find_by(id: params[:id])
+    if movie.nil?
+      render status: :not_found
+
+    else
+
+      render json:  movie.as_json(only: [:title, :overview, :release_date, :inventory]), status: :ok
+    end
+  end
+
+    def create
+      @movie = Movie.new(movie_params)
+      @movie.save!
```

Comment:
You should return a `json` response if the move wasn't able to be saved.  Provide error messages.